doc/source/examples/sharp-immersed-boundary/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst

```diff
@@ -106,7 +106,7 @@ We define the boundary conditions in order to have an inlet velocity of :math:`1
     end
 
 .. note::
-	Since using a `deal.ii mesh <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_, the boundary ``id = 1`` is by default the second boundary in the *x* axis, hence the free boundary in this problem. This is why all boundary indices are shifted.
+    Since using a `deal.ii mesh <https://www.dealii.org/current/doxygen/deal.II/namespaceGridGenerator.html>`_, the boundary ``id = 1`` is by default the second boundary in the *x* axis, hence the free boundary in this problem. This is why all boundary indices are shifted.
```

Comment:
Is this really necessary?

Reply:
I think we should keep it. It is not necessary but I think someone skipping straight to this example would be wondering why.